spec/systems/users/sign_up_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Sign up', type: :system do
+  context 'given VALID parameters' do
+    it 'logs user in and redirects to the root path' do
+      visit(new_user_registration_path)
+      fill_in('user[email]', with: 'abc@gmail.com')
+      fill_in('user[password]', with: '123456')
+      fill_in('user[password_confirmation]', with: '123456')
+      click_button(I18n.t('auth.sign_up'))
+
+      expect(page).to(have_current_path(root_path))
```

Comment:
Do we have any notice on this screen like `Sign up successfully` message, if yes, we can expect it is displaying here, what do you think @Goose97

Reply:
Fixed in e0412cf20118877470bb72ca1c246319d835d62f